tests/library/beforeunload.spec.ts

```diff
@@ -89,3 +89,24 @@ it('should access page after beforeunload', async ({ page, server }) => {
   await page.evaluate(() => document.title);
 });
 
+it('should not stall on evaluate when dismissing beforeunload', async ({ page, server }) => {
```

Comment:
can we have an issue annotation? 

https://github.com/microsoft/playwright/issues/23141